exercises/robot-simulator/tests/Tests.elm

```diff
@@ -15,46 +15,51 @@ tests =
              in
                 [ test "coordinates" <|
                     \() -> Expect.equal { x = 0, y = 0 } robot.coordinates
-                , test "bearing" <|
-                    \() -> Expect.equal North robot.bearing
+                , skip <|
+                    test "bearing" <|
+                        \() -> Expect.equal North robot.bearing
                 ]
             )
         , describe "setup"
             (let
                 robot =
                     Robot South { x = -1, y = 1 }
              in
-                [ test "coordinates" <|
-                    \() -> Expect.equal { x = -1, y = 1 } robot.coordinates
-                , test "bearing" <|
-                    \() -> Expect.equal South robot.bearing
+                [ skip <|
+                    test "coordinates" <|
+                        \() -> Expect.equal { x = -1, y = 1 } robot.coordinates
+                , skip <|
+                    test "bearing" <|
+                        \() -> Expect.equal South robot.bearing
                 ]
             )
         , describe "turn right"
             ((List.range 1 3)
                 |> List.scanl (\_ r -> turnRight r) defaultRobot
                 |> List.map .bearing
                 |> assertionList [ North, East, South, West ]
-                |> List.indexedMap (\i e -> test ("step " ++ toString i) (\() -> e))
+                |> List.indexedMap (\i e -> skip <| test ("step " ++ toString i) (\() -> e))
```

Comment:
I'm thinking this skip might be better at the describe level. elm-test won't let them forget it, but I think it'll be a bit more clear there. Also see the next describe block.